apps/documentation/docs/set-up-sls-mentor/configure-sls-mentor.md

```diff
@@ -37,6 +37,7 @@ The rule name should be one of the following list:
 - noMonoPackage
 - noProvisionedConcurrency
 - noSharedIamRoles
+- noTimeout
```

Comment:
title of rule is intended desired state, your rule should be `hasTimeout` rather than `noTimeout`. This is similar to `noSharedIamRoles` that aims at not having shared IAM roles :)